lib/src/default_index/revset_engine.rs

```diff
@@ -1346,7 +1353,7 @@ fn to_file_content(path: &RepoPath, value: MaterializedTreeValue) -> BackendResu
         MaterializedTreeValue::Symlink { id: _, target } => Ok(target.into_bytes()),
         MaterializedTreeValue::GitSubmodule(_) => Ok(vec![]),
         MaterializedTreeValue::FileConflict { contents, .. } => {
-            Ok(materialize_merge_result_to_bytes(&contents).into())
+            Ok(materialize_merge_result_to_bytes(&contents, conflict_marker_style).into())
```

Comment:
nit: move the hardcoded value here. This is also a temporary hack.

Reply:
Ok, makes sense!